src/MaterialsDesigner.jsx

```diff
@@ -1,5 +1,5 @@
+import { ThemeProvider } from "@material-ui/core/styles";
```

Comment:
Due to an overabundance of caution, can we also use default exports for `@material-ui/core` here and elsewhere too?

Reply:
No, I checked this option for `ThemeProvider` and `createTheme`, and there is no default exports for both of them.

Reply:
OK, we will just have to watch for performance issues. We should address the import sorting but it probably makes sense as a different task like `js -> jsx` because it is easier to trust programmatic updates when there is no human interaction :)

Reply:
Yes, I agree.

Reply:
Okay, I will rebase the branch again.

Reply:
Wait a minute, please. Need to update package-lock.